client/components/foldable-card/index.js

```diff
@@ -0,0 +1,161 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+ import PropTypes from 'prop-types';
+ import React, { Component } from 'react';
+ import classNames from 'classnames';
+ import { localize } from 'i18n-calypso';
+ import { noop } from 'lodash';
+
+ /**
+  * Internal Dependencies
+  */
+ import { Card } from '@wordpress/components';
```

Comment:
The current submodule's code uses `CompactCard` and `Card`, I changed this to use WordPress' components instead.